source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1264,9 +1291,11 @@ implementation::
                 return
 
             if isStateChangeError(error):
-                # Mark the server Unknown
-                unknown = new ServerDescription(type=Unknown, error=error, topologyVersion=topologyVersion)
-                onServerDescriptionChanged(unknown, connection pool for server)
+                # Don't mark server unknown in load balanced mode.
+                if type!=LoadBalanced
```

Comment:
nit: Add spacing to make this `if type != LoadBalanced`. Also, should there be a corresponding check in the `elif` code block?

Reply:
Made formatting change and added the logic to the `elif` block as well.